### PR TITLE
Reorder Patient Search buttons

### PR DESCRIPTION
**Pull request by shivankacker**
## Proposed Changes

- Makes patient name default
- shifts ordering of buttons to make more sense
image

@ohcnetwork/care-fe-code-reviewers

## Merge Checklist

- [ ] Add specs that demonstrate bug / test a new feature.
- [ ] Update [product documentation](https://docs.ohc.network).
- [ ] Ensure that UI text is kept in I18n files.
- [ ] Prep screenshot or demo video for changelog entry, and attach it to issue.
- [ ] Request for Peer Reviews
- [ ] Completion of QA

## Summary by CodeRabbit

- **New Features**
	- Improved search functionality for managing patients, including enhanced validation for phone number fields.
	- Retained both `phone_number` and `emergency_contact_number` as search options.

- **Bug Fixes**
	- Adjusted input validation logic to allow for empty values or values meeting minimum length requirements.

- **Style**
	- Minor layout adjustments to ensure a responsive and user-friendly interface.

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -119,6 +119,21 @@
         "node": ">=22.11.0"
       }
     },
+    "apps/care_scribe_fe": {
```
Comment (Jacobjeevan) [inline package-lock.json]: Are the changes in this file needed for this PR 🤔

Comment (shivankacker) [reply package-lock.json]: Whoops forgot to recheck files, thanks

**Review by rithviknishad — APPROVED** [review]
other than lockfile lgtm

**Comment by nihal467** [comment]
LGTM

**Pull request merged**